### ISSUE TITLE
a bunch of errors

### ISSUE THREAD

**Issue by CGQAQ** [Cammisuli/monodon#33]
<details><summary>nx.json</summary>
<p>

```json
{
  "targetDefaults": {
    "build": {
      "cache": true,
      "dependsOn": [
        "^build"
      ]
    },
    "lint": {
      "cache": true
    },
    "test": {
      "cache": true
    },
    "e2e": {
      "cache": true
    }
  },
  "installation": {
    "version": "17.1.3"
  },
  "plugins": [
    "@monodon/rust"
  ]
}

``` 

</p>
</details> 

<details><summary>project.json</summary>
<p>

```json
{
    "$schema": "../.nx/installation/node_modules/nx/schemas/project-schema.json",
    "name": "b3-local-server",
    "projectType": "library",
    "targets": {
        "build": {
            "executor": "@monodon/rust:build",
            "outputs": [
                "{options.target-dir}"
            ],
            "options": {
                "target-dir": "dist/target/b3-local-server"
            },
            "configurations": {
                "production": {
                    "release": true
                }
            }
        },
        "test": {
            "executor": "@monodon/rust:test",
            "outputs": [
                "{options.target-dir}"
            ],
            "options": {
                "target-dir": "dist/target/b3-local-server"
            },
            "configurations": {
                "production": {
                    "release": true
                }
            }
        },
        "lint": {
            "executor": "@monodon/rust:lint",
            "outputs": [
                "{options.target-dir}"
            ],
            "options": {
                "target-dir": "dist/target/b3-local-server"
            }
        }
    }
}
``` 

</p>
</details> 

![image](https://github.com/Cammisuli/monodon/assets/15936231/29b07f50-fb65-47a8-9c67-0e0b880ee35c)
![image](https://github.com/Cammisuli/monodon/assets/15936231/ce561a94-5518-4d56-a4ba-263e865042ac)
![image](https://github.com/Cammisuli/monodon/assets/15936231/f48eae99-2a73-4faa-a5a2-a9f7c05146c3)

**Comment by Cammisuli** [comment]
Oh shoot, it looks like a lot of those dependencies didnt auto populate in the build of package.json. It works in my e2e because nx is already there and has these deps already. 

I'll fix it

**Comment by Cammisuli** [comment]
fixed with 1e0166713d058d159cd13ca7fd343f7191b76a35

**Comment by Cammisuli** [comment]
fixed with 1e0166713d058d159cd13ca7fd343f7191b76a35
